ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -375,9 +386,18 @@ export const WorkspaceEdit = fp.flow(withCurrentWorkspace(), withCdrVersions(),
     }
 
     async fetchBillingAccounts() {
+      this.setState({fetchBillingAccountLoading: true});
       const billingAccounts = (await userApi().listBillingAccounts()).billingAccounts;
+
+      // Replace the free billing account with a new display name that has spend usage.
+      const displayFreeTierAccount: BillingAccount = {
+        ...billingAccounts.find(billingAccount => billingAccount.isFreeTier),
+        displayName: this.formatFreeTierBillingAccountName()
+      }
+      const displayBillingAccounts = billingAccounts.map(b => b.isFreeTier ? displayFreeTierAccount : b)
```

Comment:
nit: rather than a find, then map, it is probably cleaner to just do the conversion in the map:

```
= ..map((b) => {
  if (b.isFreeTier) {
    return {
        ...b,
        displayName: this.formatFreeTierBillingAccountName()
     };
  }
  return b;
});
```